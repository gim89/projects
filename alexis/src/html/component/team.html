<section class="team section__padding section__text">
  <div class="container">
    <h2 class="team__title">Our Team.</h2>
    <p class="team__subtitle subtitle__margin">
      We understand your requirement and provide quality works.
    </p>
    <div class="team__tabs">
      <div class="team__nav section__body">
        <a href="#panel-1" class="team__link team__link--after team__link--active">
          <section class="team__item team__item--active">
            <img class="team__img" src="img/team-one.jpg" alt="team-one-img">
            <h4 class="team__caption color__red">Michail Schmitt</h4>
              <p class="team__text">Founder &amp; CEO</p>
          </section>
          <div class="line background__red"></div>
        </a>
        <a href="#panel-2" class="team__link team__link--after">
          <section class="team__item team__item--active">
            <img class="team__img" src="img/team-two.jpg" alt="team-one-img">
            <h4 class="team__caption color__green">David Kramer</h4>
              <p class="team__text">Lead Developer</p>
          </section>
          <div class="line background__green"></div>
        </a>
        <a href="#panel-3" class="team__link team__link--after">
          <section class="team__item team__item--active">
            <img class="team__img" src="img/team-three.jpg" alt="team-one-img">
            <h4 class="team__caption color__purple">Anna Schroder</h4>
              <p class="team__text">Designer</p>
          </section>
          <div class="line background__purple"></div>
        </a>
        <a href="#panel-4" class="team__link team__link--after">
          <section class="team__item team__item--active">
            <img class="team__img" src="img/team-four.jpg" alt="team-one-img">
            <h4 class="team__caption color__orange">Jason Teichmann</h4>
              <p class="team__text">Consultant</p>
          </section>
          <div class="line background__orange"></div>
        </a>
      </div>
      <ul class="team__content">
        <li class="team__panel team__panel--active" id="panel-1">
          <article class="team__article">
            <h3 class="team__caption">Michail Schmitt</h3>
            <p class="team__info">
              Monetization first mover advantage iPad. Interaction design buzz handshake. Stealth agile development branding. Backing facebook
              hackathon conversion startup scrum project niche market business-to-consumer paradigm shift traction. Rockstar
              bandwidth twitter ownership success disruptive series A financing user experience buyer client social proof
              validation hypotheses. User experience crowdfunding facebook seed round A/B testing mass market.
            </p>
          </article>
          <img class="team__image" src="img/team-image.png" alt="team-image">
          <div class="team__social">
            <i class="team__soc">
              <span class="team__icon  team__fb"></span>
              <a href="#" class="team__link team__link--hover">Facebook</a>
            </i>
            <i class="team__soc">
              <span class="team__icon team__tw"></span>
              <a href="#" class="team__link team__link--hover">Twitter</a>
            </i>
            <i class="team__soc">
              <span class="team__icon team__in"></span>
              <a href="#" class="team__link team__link--hover">LinkedIn</a>
            </i>
            <i class="team__soc">
              <span class="team__icon team__inst"></span>
              <a href="#" class="team__link team__link--hover">Instagram</a>
            </i>
          </div>
        </li>
        <li class="team__panel" id="panel-2">
          <article class="team__article">
            <h3 class="team__caption">David Kramer</h3>
            <p class="team__info">
              Monetization first mover advantage iPad. Interaction design buzz handshake. Stealth agile development branding. Backing facebook
              hackathon conversion startup scrum project niche market business-to-consumer paradigm shift traction. Rockstar
              bandwidth twitter ownership success disruptive series A financing user experience buyer client social proof
              validation hypotheses. User experience crowdfunding facebook seed round A/B testing mass market.
            </p>
          </article>
          <img class="team__image" src="img/team-image.png" alt="team-image">
          <div class="team__social">
            <i class="team__soc">
              <span class="team__icon  team__fb"></span>
              <a href="#" class="team__link team__link--hover">Facebook</a>
            </i>
            <i class="team__soc">
              <span class="team__icon team__tw"></span>
              <a href="#" class="team__link team__link--hover">Twitter</a>
            </i>
            <i class="team__soc">
              <span class="team__icon team__in"></span>
              <a href="#" class="team__link team__link--hover">LinkedIn</a>
            </i>
            <i class="team__soc">
              <span class="team__icon team__inst"></span>
              <a href="#" class="team__link team__link--hover">Instagram</a>
            </i>
          </div>
        </li>
        <li class="team__panel" id="panel-3">
          <article class="team__article">
            <h3 class="team__caption">Anna Schroder</h3>
            <p class="team__info">
              Monetization first mover advantage iPad. Interaction design buzz handshake. Stealth agile development branding. Backing facebook
              hackathon conversion startup scrum project niche market business-to-consumer paradigm shift traction. Rockstar
              bandwidth twitter ownership success disruptive series A financing user experience buyer client social proof
              validation hypotheses. User experience crowdfunding facebook seed round A/B testing mass market.
            </p>
          </article>
          <img class="team__image" src="img/team-image.png" alt="team-image">
          <div class="team__social">
            <i class="team__soc">
              <span class="team__icon  team__fb"></span>
              <a href="#" class="team__link team__link--hover">Facebook</a>
            </i>
            <i class="team__soc">
              <span class="team__icon team__tw"></span>
              <a href="#" class="team__link team__link--hover">Twitter</a>
            </i>
            <i class="team__soc">
              <span class="team__icon team__in"></span>
              <a href="#" class="team__link team__link--hover">LinkedIn</a>
            </i>
            <i class="team__soc">
              <span class="team__icon team__inst"></span>
              <a href="#" class="team__link team__link--hover">Instagram</a>
            </i>
          </div>
        </li>
        <li class="team__panel" id="panel-4">
          <article class="team__article">
            <h3 class="team__caption">Jason Teichmann</h3>
            <p class="team__info">
              Monetization first mover advantage iPad. Interaction design buzz handshake. Stealth agile development branding. Backing facebook
              hackathon conversion startup scrum project niche market business-to-consumer paradigm shift traction. Rockstar
              bandwidth twitter ownership success disruptive series A financing user experience buyer client social proof
              validation hypotheses. User experience crowdfunding facebook seed round A/B testing mass market.
            </p>
          </article>
          <img class="team__image" src="img/team-image.png" alt="team-image">
          <div class="team__social">
            <i class="team__soc">
              <span class="team__icon  team__fb"></span>
              <a href="#" class="team__link team__link--hover">Facebook</a>
            </i>
            <i class="team__soc">
              <span class="team__icon team__tw"></span>
              <a href="#" class="team__link team__link--hover">Twitter</a>
            </i>
            <i class="team__soc">
              <span class="team__icon team__in"></span>
              <a href="#" class="team__link team__link--hover">LinkedIn</a>
            </i>
            <i class="team__soc">
              <span class="team__icon team__inst"></span>
              <a href="#" class="team__link team__link--hover">Instagram</a>
            </i>
          </div>
        </li>
      </ul>
    </div>
  </div>
</section>