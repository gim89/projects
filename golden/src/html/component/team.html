<section class="team section__padding">
  <div class="container">
    <h2 class="team__title">our amazing team</h2>
    <p class="team__subtitle subtitle__margin">Lorem ipsum dolor sit amet consectetur.</p>
    <div class="team__body">
      <div class="team__item">
        <img src="img/team.jpg" alt="" class="team__img">
        <h3 class="team__caption">Kimberly Thompson</h3>
        <p class="team__span">Marketer</p>
        <div class="team__social">
          <div class="twitter twitter--hover social__icon"></div>
          <div class="facebook facebook--hover  social__icon"></div>
          <div class="pinterest pinterest--hover  social__icon"></div>
        </div>
      </div>
      <div class="team__item">
        <img src="img/team.jpg" alt="" class="team__img">
        <h3 class="team__caption">Rico Massimo</h3>
        <p class="team__span">Coder</p>
        <div class="team__social">
          <div class="twitter twitter--hover social__icon"></div>
          <div class="facebook facebook--hover social__icon"></div>
          <div class="google google--hover social__icon"></div>
        </div>
      </div>
      <div class="team__item">
        <img src="img/team.jpg" alt="" class="team__img">
        <h3 class="team__caption">Uku Mason</h3>
        <p class="team__span">Graphic Designer</p>
        <div class="team__social">
          <div class="facebook facebook--hover social__icon"></div>
          <div class="pinterest pinterest--hover social__icon"></div>
        </div>
      </div>
    </div>
    <p class="team__text">Lorem ipsum dolor sit amet consectetur adipisicing elit. Laudantium excepturi, dignissimos tempore nisi et maxime officiis tempora? Eligendi, architecto minus?</p>
  </div>
</section>